src/ngraph/builder/autobroadcast.cpp

```diff
@@ -249,8 +249,8 @@ namespace ngraph
             // Handle the trivial case...
             if (arg1_in_shape == arg2_in_shape)
             {
-                return make_pair(args.first.as_single_output_node(),
-                                 args.second.as_single_output_node());
+                return make_pair(args.first.get_node_shared_ptr(),
```

Comment:
I suspect this should be returning `make_pair(args.first, args.second)` since those args aren't necessarily using 1-output nodes.

Reply:
I changed the method to return pair<Output<Node>, Output<Node>> instead of a pair of nodes.